server/models/user.js

```diff
@@ -73,9 +73,11 @@ module.exports = (sequelize, DataTypes) => {
       foreignKey: 'followerId',
       as: 'follower',
     });
-    User.hasMany(Article, {
+
+    User.belongsToMany(Article, {
```

Comment:
It's all cool to this point as I don't understand why a many-many association is used here. Can you explain further to me?